layers/+lang/javascript/packages.el

```diff
@@ -191,6 +191,9 @@
     :init (add-hook 'js2-mode-hook 'tern-mode)
     :config
     (progn
+      (if javascript-disable-tern-port-files
+          (setq tern-command (append tern-command '("--no-port-file")))
+        nil)
```

Comment:
Cleaner: `(when javascript-disable-tern-port-files (add-to-list 'term-command "--no-port-file" 'append))`